pkg/actions/nodegroup/create.go

```diff
@@ -30,6 +30,7 @@ type CreateOpts struct {
 	InstallNeuronDevicePlugin bool
 	InstallNvidiaDevicePlugin bool
 	DryRun                    bool
+	CheckOutdatedAddons       bool
```

Comment:
since we want the default behaviour to be checking the addons, shall we change this to `SkipOutdatedAddonsCheck` or something similar, so we can rely on the defaulting to `false` to preserve the existing behaviour :smile: